### PR TITLE
Add details for GKE Autopilot compute class + cleanup

### PR DESCRIPTION
**Pull request by JacksonDavenport**
### What does this PR do? What is the motivation?

GKE Autopilot supports compute classes for dedicated hardware, and like their GKE Spot Pods, sort of silently manages these with Tainted nodes. Autopilot will automatically add Tolerations to the workloads you specify, however, you need to do this manually to the Agent. 

https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-compute-classes

Additionally does some minor cleanup and removes the Operator steps for Spot Pods, since the Operator does not support Autopilot anyway. 

From customer requests.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/containers/kubernetes/distributions.md [hunk 1]
```diff
@@ -330,7 +313,22 @@ agents:
 {{% /tab %}}
 {{< /tabs >}}
 
-**Note**: Network Performance Monitoring is not supported for GKE Autopilot.
+Similarly when using [GKE Autopilot Compute classes][11] to run workloads that have specific hardware requirements, take note of the [Taints][9] that GKE Autopilot is applying to these specific nodes and add matching Tolerations to the Agent DaemonSet. You can match the Tolerations on your corresponding pods. For example for the `Scale-Out` compute class use a Toleration like:
```
Comment (aliciascott) [inline content/en/containers/kubernetes/distributions.md:316]:
```suggestion
Similarly when using [GKE Autopilot Compute classes][11] to run workloads that have specific hardware requirements, take note of the [Taints][9] that GKE Autopilot is applying to these specific nodes and add matching tolerations to the Agent DaemonSet. You can match the tolerations on your corresponding pods. For example for the `Scale-Out` compute class use a toleration like:
```

Comment (aliciascott) [reply content/en/containers/kubernetes/distributions.md:316]: Is this generally industry accepted as capitalized? If not should probably use lower-case

File: content/en/containers/kubernetes/distributions.md [hunk 2]
```diff
@@ -292,30 +294,11 @@ providers:
 {{% /tab %}}
 {{< /tabs >}}
 
-### Spot pods and instances
+### Spot pods and compute Classes
```
Comment (aliciascott) [inline content/en/containers/kubernetes/distributions.md:297]:
```suggestion
### Spot pods and compute classes
```
I see "classes" is lower-case below, we should also lowercase here to adhere to sentence case for headings

**Review by aliciascott — APPROVED** [review]
Hi @JacksonDavenport went ahead and approved this as the changes requested are non-blocking. Note you can now add a comment to approved docs PR's with just a comment with `/merge` and go ahead and merge this when you are ready!

File: content/en/containers/kubernetes/distributions.md (hunk 2 above)
Comment (JacksonDavenport) [reply content/en/containers/kubernetes/distributions.md:297]: Oops I remembered to lower "compute" but forgot that.

File: content/en/containers/kubernetes/distributions.md (hunk 1 above)
Comment (JacksonDavenport) [reply content/en/containers/kubernetes/distributions.md]: Good call out, I thought K8s did so I mirrored, but I must have skimmed to quickly. Removed those. 
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/

**Pull request merged**